src/main/python/model/minidsp.py

```diff
@@ -449,7 +449,7 @@ def overwrite(self, filters, target, metadata=None):
                                                                   show_index=False, to_hex=True,
                                                                   fixed_point=self.__is_fixed_point_hardware())[0]
                                     child.find('hex').text = f"{hex_txt},"
-        if metadata is not None:
```

Comment:
isn't it just `if metadata is True:`

Reply:
I see what is going on now, QAction.triggered has a kwarg checked which will be a boolean (False by default) so this is explicitly trapping that situation because that boolean overrides

`def export_beq_filter(self, metadata=None):`

in app.py

it would be simpler/more direct/less confusing to change this in export_beq_filter, I'm not even sure why that has a kwarg when it is only ever called by the action

Reply:
It's not `if metadata is true` unless Python has a feature I don't know about.  `metadata` isn't a bool coming in.  It's a dictionary containing all of the key:value pairs to be added to the xml.  So it needs to drop down to where the xml is manipulated.

Reply:
2nd comment explains (what is IMV) a cleaner fix

Reply:
I created a new PR for the change since I redid my branch when cleaning up the commits.  

I fixed the export_beq_filter method like you said.  When I looked at the code it made sense.  I had an old version of it in my head.  Originally I had used the export_beq_filter method in another location and was thinking it was still used there.